contents/docs/apps/snowflake-export.md

```diff
@@ -4,72 +4,205 @@ layout: app
 github: https://github.com/PostHog/snowflake-export-plugin
 installUrl: https://app.posthog.com/project/apps?name=Snowflake
 thumbnail: ../../apps/thumbnails/snowflake.svg
+official: true
 topics:
     - snowflake-export
 ---
 
-### What does the Snowflake Export app do?
+This app allows you to export both live and historical events from PostHog into Snowflake.
+This is useful when you want to run custom SQL queries on your data in PostHog using Snowflake's high-performance infrastructure.
 
-This app uses a Snowflake external stage to stage events in object storage - Amazon S3 or Google Cloud Storage. Staged events (stored in object storage as files containing event batches) are then copied into the final destination – your Snowflake table – once an hour.
+This app utilizes a Snowflake [external stage](https://docs.snowflake.com/en/sql-reference/sql/create-stage.html) to stage events in object storage - Amazon S3 or Google Cloud Storage.
+Staged events (stored in object storage as files containing event batches) are then copied into the final destination – your Snowflake table – once an hour.
 
-### What are the requirements for this app?
+### Install
 
-The First Time Event Tracker requires either PostHog Cloud, or a self-hosted PostHog instance running [version 1.30.0](https://posthog.com/blog/the-posthog-array-1-30-0) or later.
+#### PostHog Cloud
 
-Not running 1.30.0? Find out [how to update your self-hosted PostHog deployment](https://posthog.com/docs/self-host/configure/upgrading-posthog)!
+PostHog Cloud users can find the app [here in their dashboard](https://app.posthog.com/project/apps?name=Snowflake).
```

Comment:
"dashboard" might be somewhat confusing here, as we say "dashboards" for collections of insights. Would probably be clearer to say "on the Apps page"